utils/datasets_util.py

```diff
@@ -0,0 +1,259 @@
+import numpy as np
+import os
+import os.path as op
+import glob
+import h5py
+import tensorflow as tf
+import tensorflow_datasets as tfds
+
+
+USPS_PATH = "/home/dbash_google_com/datasets/usps.h5"
+
+def hdf5(path, data_key="data", target_key="target"):
+  """
+      loads data from hdf5:
+      - hdf5 should have 'train' and 'test' groups
+      - each group should have 'data' and 'target' dataset or spcify the key
+      - flatten means to flatten images N * (C * H * W) as N * D array
+  """
+  with h5py.File(path, "r") as hf:
+    train = hf.get("train")
+    x_tr = train.get(data_key)[:]
+    y_tr = train.get(target_key)[:]
+    test = hf.get("test")
+    x_te = test.get(data_key)[:]
+    y_te = test.get(target_key)[:]
+    x_tr = x_tr.reshape((x_tr.shape[0], 16, 16, 1))*255.
+    x_te = x_te.reshape((x_te.shape[0], 16, 16, 1))*255.
+  return x_tr, y_tr, x_te, y_te
+
+def get_usps_tf_dataset(path):
+  """returns tf.data.Dataset object with USPS dataset.
+
+  Arguments:
+  path: path to h5 file."""
```

Comment:
This is just a question of taste, but I would generally format a doc as follows:

```suggestion
  """Returns tf.data.Dataset object with USPS dataset.

  Arguments:
    path: path to h5 file.
"""
```